.github/workflows/dotnet.yml

```diff
@@ -1,11 +1,28 @@
 name: .Net
 on:
   push:
+    tags:
+    - RELEASE
     branches:
     - master
   pull_request:
     branches:
     - master
+env:
+  IS_RELEASE_CANDIDATE: >-
+    ${{
+      (
+        github.event_name == 'pull_request' &&
+        startsWith(github.event.pull_request.title, 'RELEASES:') &&
+        contains(github.event.pull_request.labels.*.name, 'RELEASES')
```

Comment:
```suggestion
        contains(github.event.pull_request.labels.*.name, 'RELEASE')
```

Reply:
The issue mentions checking for `RELEASES` label, should we checf for `RELEASE` instead?